src/plugins/okta/login.ts

```diff
@@ -76,7 +80,17 @@ const fetchSamlResponse = async (
 
 /** Logs in to Okta via OIDC */
 export const oktaLogin = async (org: OrgData) =>
-  oidcLogin(org, "openid email profile okta.apps.sso");
+  oidcLogin<AuthorizeResponse, TokenResponse>(
+    oidcLoginSteps(org, "openid email profile okta.apps.sso", () => {
+      if (org.providerType !== "okta") {
+        throw `Invalid provider type ${org.providerType}`;
```

Comment:
What is the user supposed to do with this information? Or does hitting this imply a code bug?

If it's the latter, we should emit a stack trace:
```suggestion
        throw new Error(`Invalid provider type ${org.providerType} (expected "okta")`);
```